### ISSUE TITLE
Clean and improve sdk/math.ooc

### ISSUE THREAD

**Issue by marcusnaslund** [cogneco/ooc-kean#706]
There are a number of things to do.

- [x] Fix indentation and make it magic compliant (not necessary now, but will be later)
- [x] Remove global functions, instead make sure we use methods defined as extensions on `Int`, `Float` and so on.
- [x] Remove the global variables `PI`, `constantPi` and so on, instead use them as extensions.
- [x] ~~Move the `Range` extension and make it a part of `Range` in `sdk/lang/Numbers.ooc`~~
- [x] Fix inconsistencies and errors with `%` and `modulo` (see #508)
- [x] Change static methods into regular methods. Some can probably be properties.

Eventually, we want to make this a part of `ooc-math` and thus get rid of the `import math` line altogether. The above will help a great deal and will need to be done first.

**Comment by marcusnaslund** [comment]
Let's keep `sin`, `cos`, `tan`, `asin`, `acos`, `atan` as global functions? How about `sqrt`? Also, should we change them to their `Float` counterparts since that's what we mainly use? @niklasborwell @sebastianbaginski

**Comment by sebastianbaginski** [comment]
I think you know what is my opinion
```ooc
sin(x * pow(Math pi, 2)) * atan(pow(y, 3))
```
vs.
```ooc
(x * Math pi pow(2)) sin() * y pow(3) atan()
```
first one is much easier to read, at least for me. It is just the natural format, which we all know from math classes.

**Comment by marcusnaslund** [comment]
`y pow(3)` looks good to me though. But I see your point about `sin`, `cos` etc. and I'm OK with keeping them. Right now they take and return `Double`, though.

**Comment by niklasborwell** [comment]
Yeah it would make sense to have float functions instead.

**Comment by sebastianbaginski** [comment]
... not mentioning the fact that everyone will need to remember to write `(-5) pow(2)` instead of `-5 pow(2)` :)

**Comment by marcusnaslund** [comment]
Fixed by #1172
